templates/hub/hub.css

```diff
@@ -72,6 +84,10 @@
   background-color: var(--udexColorBlue2);
 }
 
+.section.background-blue-7 {
+  background-color: var(--udexColorBlue7);
```

Comment:
needs to also set text color to white

Reply:
The text color should be set independently, they should not be combined